src/Symfony/Component/Cache/Tests/Adapter/PdoDbalAdapterTest.php

```diff
@@ -43,4 +47,125 @@ public function createCachePool(int $defaultLifetime = 0): CacheItemPoolInterfac
     {
         return new PdoAdapter(DriverManager::getConnection(['driver' => 'pdo_sqlite', 'path' => self::$dbFile]), '', $defaultLifetime);
     }
+
+    public function testConfigureSchemaDecoratedDbalDriver()
+    {
+        $connection = DriverManager::getConnection(['driver' => 'pdo_sqlite', 'path' => self::$dbFile]);
+        if (interface_exists(\Doctrine\DBAL\Driver\Middleware::class)) {
+            $middleware = $this->createMock(\Doctrine\DBAL\Driver\Middleware::class);
+            $middleware
+                ->method('wrap')
+                ->willReturn(new DriverWrapperV3($connection->getDriver()));
+
+            $config = new Configuration();
+            $config->setMiddlewares([$middleware]);
+
+            $connection = DriverManager::getConnection(['driver' => 'pdo_sqlite', 'path' => self::$dbFile], $config);
+        } else {
+            $reflectionProperty = new \ReflectionProperty($connection, '_driver');
+            $reflectionProperty->setAccessible(true);
+            $reflectionProperty->setValue($connection, new DriverWrapperV2($reflectionProperty->getValue($connection)));
+            $reflectionProperty->setAccessible(false);
+        }
+
+        $adapter = new PdoAdapter($connection);
+        $adapter->createTable();
+
+        $item = $adapter->getItem('key');
+        $item->set('value');
+        $this->assertTrue($adapter->save($item));
+    }
+}
+
+if (interface_exists(\Doctrine\DBAL\Driver\Middleware::class)) {
+    class DriverWrapperV3 implements Driver
```

Comment:
Let‘s move the class into a dedicated file in the `Tests\Fixtures` namespace, so we don‘t need a conditional declaration.

Reply:
It seems `php .github/patch-types.php` is not happy with this class existing as fixture: https://github.com/symfony/symfony/pull/42011/checks?check_run_id=3016796261 (Or is something else wrong?)

Reply:
patch-types has a list of files to exclude. Please add your fixture there.

Reply:
Thanks